### PR TITLE
skip IaaS and IAM/SEC Calls on Sept 4th due to travels

### PR DESCRIPTION
**Pull request by fkr**
discussed with @gtema (among others)

### CODE REVIEW

File: teams/iam_n_sec/team_meetings.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ events:
         - 2024-05-01 11:35:00
         - 2024-05-15 11:35:00
         - 2024-07-10 11:35:00
+        - 2024-08-04 11:35:00
```
Comment (gtema) [inline teams/iam_n_sec/team_meetings.yml:31]: should be September and not Augist

**Review by gtema — CHANGES_REQUESTED** [review]

File: teams/iam_n_sec/team_meetings.yml (hunk 1 above)
Comment (jschoone) [inline teams/iam_n_sec/team_meetings.yml:31]:
```suggestion
        - 2024-09-04 11:35:00
```

Comment (garloff) [reply teams/iam_n_sec/team_meetings.yml]: Resolved by Jan's suggestion in commit https://github.com/SovereignCloudStack/calendar/pull/308/commits/5c23627714211be8158fef73a9f01bd353f03ba1

**Review by garloff — APPROVED** [review]
/lgtm

**Review by gtema — APPROVED** [review]

**Review by jschoone — APPROVED** [review]

**Pull request merged**